definitions/defaults/global.lua

```diff
@@ -56,7 +57,7 @@ return {
     ["<C-'>"] = "DeleteMark",
     ["<C-r>"] = "Redo",
     ["u"] = "Undo",
-    ["R"] = "RecordOrStop",
+    ["R"] = "RecordToggle",
```

Comment:
It feels like this might yield a smoother recording experience - what do you think?

Reply:
I just saw that the action already exists so this could be just changed to ToggleRecord.

Reply:
I changed this because I thought it feelt smoother if I could stop recording whilst keeping the playback.

Reply:
I am also going to see if I can maybe prevent ESC from stopping playback because this could maybe make it possible to use RK to create more of a live experience. I don’t like when playback stops and I didn’t explocitly tellbit to do so.

Reply:
I think it's good that way, one can just press enter if they want it the previous way.

" Do you have any tips on how I can add
new action commands faster? How do you do it?"
I copy the reaper action id from the action list, and plop it in my text editor, give it a name, and that's all! 

"I am also going to see if I can maybe prevent ESC fr"
That's a good point about ESC stopping live experience, feel free to take that out by altering the action.